src/lightning/__version__.py

```diff
@@ -1 +1,18 @@
-version = "1.8.0"
+# in case you have not installed the package and have only set python path to this source
+
+import os
+
+_PATH_PKG = os.path.dirname(__file__)
+_PATH_VER = os.path.join(_PATH_PKG, "version.info")
+version: str
+
+if os.path.isfile(_PATH_VER):
+    with open(_PATH_VER, encoding="utf-8") as fo:
+        version = fo.readlines()[0].strip()
+else:
+    try:
+        import subprocess
+
+        version = subprocess.check_output(["git", "rev-parse", "--short", "HEAD"]).decode("ascii").strip()
+    except Exception:
+        version = ""
```

Comment:
I still believe that we should not have this fallback.

_PATH_VER should always exist

Reply:
So the fallback shall root to the global one?
I have set default as "unknown"